include/IECoreVDB/VDBObject.h

```diff
@@ -76,6 +76,8 @@ class IECOREVDB_API VDBObject : public IECoreScene::VisibleRenderable
 		openvdb::GridBase::ConstPtr findGrid( const std::string &name ) const;
 		openvdb::GridBase::Ptr findGrid( const std::string &name );
 
+		openvdb::GridBase::ConstPtr findMetadata( const std::string& name ) const;
```

Comment:
I find it extremely confusing that a `metadata()` method should return a grid, just with the "griddy" bits missing. I was ignoring this as long as it was just part of the private implementation, but I think it needs addressing if it's to be part of the public API. Can we return a MetaMap::ConstPtr instead? Or maybe keep this internal? I suppose even if we returned the MetaMap, it could be downcast to a grid afterwards?

Can you remind me what the motivation was for the original `CompoundObjectPtr VDBObject metadata( name )`? It seems a bit odd to have both that and `findMetadata()`, so perhaps that informs our decision somewhat?

Reply:
>  find it extremely confusing that a metadata() method should return a grid, just with the "griddy" bits missing. I was ignoring this as long as it was just part of the private implementation, but I think it needs addressing if it's to be part of the public API. Can we return a MetaMap::ConstPtr instead? Or maybe keep this internal?

I'll make it internal. 

> Can you remind me what the motivation was for the original CompoundObjectPtr VDBObject metadata( name )?

For the Scene Inspector to display information on the grids.

Reply:
17aef71 - Another revision which doesn't expose the the metadata only access.